apps/www/components/LaunchWeek/12/LWSummary.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import Image from 'next/image'
+import Link from 'next/link'
+import { buildDays, mainDays } from './Releases/data'
+
+const LW11Summary = () => {
+  const days = mainDays()
```

Comment:
does this need a check since it's not async? 

ie:  in the return doing a `days &&`